### PR TITLE
Widgets: Create a standalone image widget

### PR DESCRIPTION
**Pull request by wassgha**
This PR adds  a new widget that displays a single image. Options for the widget include adding a title, background color and choosing whether to zoom in the image or not.

### Screenshots

Options:
Screen Shot 2019-05-12 at 4 14 42 PM

Image (Zoomed in):
Screen Shot 2019-05-12 at 4 15 32 PM

Image (Fit to container):
Screen Shot 2019-05-12 at 4 15 22 PM

### CODE REVIEW

File: widgets/image/src/ImageOptions.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react'
+import { Form, Input, InlineInputGroup } from '../../../components/Form'
+import { standaloneUpload } from '../../../actions/slide'
+
+class WebOptions extends Component {
```
Comment (binicij) [inline widgets/image/src/ImageOptions.js]: Should this be changed to ImageOptions

File: widgets/image/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import BaseWidget from '../base_widget'
+import ImageContent from './src/ImageContent'
+import ImageOptions from './src/ImageOptions'
+
+export default class Web extends BaseWidget {
```
Comment (binicij) [inline widgets/image/index.js]: Should this be changed to class Image

**Review by binicij — CHANGES_REQUESTED** [review]
Looks good for the most part, just wondering if we needed to change the class names in the files I left comments in.

File: widgets/image/src/ImageOptions.js (hunk 1 above)
Comment (wassgha) [reply widgets/image/src/ImageOptions.js]: Fixed

File: widgets/image/index.js (hunk 2 above)
Comment (wassgha) [reply widgets/image/index.js]: Fixed

**Review by binicij — APPROVED** [review]

**Pull request merged**